tests/loggers/test_tensorboard.py

```diff
@@ -1,9 +1,48 @@
 from argparse import Namespace
 
+import os
 import pytest
 import torch
 
+import tests.base.utils as tutils
+from pytorch_lightning import Trainer
+from pytorch_lightning.callbacks import ModelCheckpoint
 from pytorch_lightning.loggers import TensorBoardLogger
+from tests.base import EvalModelTemplate
+from tensorboard.backend.event_processing.event_accumulator import EventAccumulator
+import yaml
+
+
+@pytest.importorskip('torch', minversion='1.5.0')
```

Comment:
@Borda is this correct? can you modify so we can merge

Reply:
it seems it is not among marks so we need to do standard `skipif`
https://docs.pytest.org/en/stable/reference.html#marks